static/app/routes.tsx

```diff
@@ -1688,7 +1688,7 @@ function buildRoutes() {
       <IndexRoute component={make(() => import('sentry/views/profiling/content'))} />
       <Route
         path="onboarding/"
-        component={make(() => import('sentry/views/profiling/onboarding'))}
+        component={make(() => import('sentry/views/profiling/legacyOnboarding'))}
```

Comment:
This going to break the current setup onboarding button temporarily. Let's not rename it until the new one is ready.

Reply:
Why would that be? Did I forget to update something? It should still be the /onboarding route, but just load the different file

Reply:
mistook it for the url, this looks good 👍